module/dice/d20-roll.mjs

```diff
@@ -16,10 +16,7 @@
 export default class D20Roll extends Roll {
   constructor(formula, data, options) {
     super(formula, data, options);
-    if ( !((this.terms[0] instanceof Die) && (this.terms[0].faces === 20)) ) {
-      throw new Error(`Invalid D20Roll formula provided ${this._formula}`);
-    }
-    if ( !this.options.configured ) this.configureModifiers();
+    if ( this.#validD20Roll && !this.options.configured ) this.configureModifiers();
```

Comment:
Given that `configureModifiers` checks for `#validD20Roll`, I don't think we also need to guard it here.